ospfd/ospf_vty.c

```diff
@@ -5065,6 +5065,7 @@ static void show_ip_ospf_neighbor_detail_sub(struct vty *vty,
 	json_object *json_neigh = NULL, *json_neigh_array = NULL;
 	char neigh_str[INET_ADDRSTRLEN] = {0};
 	char neigh_state[16] = {0};
+	struct ospf_neighbor *nbr1;
```

Comment:
nit: small suggestion, please use some meaningful name instead of nbr1 , like ospfDrNbr/ OspfBdrNbr

Reply:
camel form variable names is not good idea. I've changed to nbr_dr and nbr_bdr.